spec/builtin/web/spider_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+require 'ronin/recon/builtin/web/spider'
+require 'uri'
+
+describe Ronin::Recon::Web::Spider do
+  describe "#process" do
```

Comment:
Since these specs are accessing the network, they should be tagged with `:network` so the specs can be ran without network access.

Or we should use `webmock/rspec` and `Sinatra` to funnel all requests to `example.com` to a fake `Sinatra` app that we control.

* https://github.com/postmodern/spidr/blob/5ab77427ab28b6df0c07d49ddb2bc316edee7547/spec/agent_spec.rb#L136-L185
* https://github.com/postmodern/spidr/blob/5ab77427ab28b6df0c07d49ddb2bc316edee7547/spec/example_app.rb + https://github.com/postmodern/spidr/blob/5ab77427ab28b6df0c07d49ddb2bc316edee7547/spec/agent_spec.rb#L458-L477